src/SocialMediaFooter.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const SocialMediaFooter = () => {
+    return (
+        <ul class="footer-social-links list-reset">
+            <li>
+                <a href="#">
+                    <span class="screen-reader-text">Facebook</span>
+                    <svg width="16" height="16" xmlns="http://www.w3.org/2000/svg">
```

Comment:
To simplify this Footer component, you can create react components containing all these icons